docs/devsguide/new_dev_guide.rst

```diff
@@ -209,5 +211,374 @@ text (Vim, TextEdit, Nano, etc.) and doesn’t invisibly add characters to the
 file (like Word). The only important part is to write the file in a manner that’s 
 considered reStructuredText (check out http://sphinx-doc.org/rest.html). Then, 
 Sphinx will do everything else under the hood as described `here 
-<http://pyne.io/devsguide/website.html>`__. Finally, commit these changes to 
-your forked version and submit a pull request.
+<http://pyne.io/devsguide/website.html>`__. Before you make the pull request, 
+make sure that you include a news file named after the purpose of your pr 
+following the template located in the news folder. Finally, commit these changes to 
+your forked version and submit a pull request. 
+
+---------------------
+Making a Pull Request
```

Comment:
I wonder if this section fits better above the section on modifying documentation?  Making a pull request is ubiquitous to contributing to PyNE and prerequisite knowledge to updating the docs.

Reply:
Also, the instruction about adding a news file might fit better under this generic info on making a pull request.

Reply:
I agree, the PR section does make more sense above the documentation section. I've moved it, and the information about the news file with my latest commit